docs/rules/require-prop-comment.md

```diff
@@ -0,0 +1,68 @@
+---
+pageClass: rule-details
+sidebarDepth: 0
+title: vue/require-prop-comment
+description: require prop should have a comment
+---
+# vue/require-prop-comment
+
+> require prop should have a comment
+
+- :exclamation: <badge text="This rule has not been released yet." vertical="middle" type="error"> ***This rule has not been released yet.*** </badge>
+
+## :book: Rule Details
+
+This rule requires that the prop should have a comment
```

Comment:
```suggestion
This rule enforces that every prop has a comment that documents it.
```

Reply:
It has been completed and will be merged in the next submission